docs/server-api-auth.md

```diff
@@ -0,0 +1,145 @@
+---
+id: server-api-auth
+title: Set up server API authorization
+sidebar_label: API auth setup
+---
+
+## Overview
+
+Temporal supports the ability to restrict API call access via an optional authorization mechanism. It enables the server to process or deny individual calls based on a caller's criteria. Criteria can include the caller's unique permissions, the name of the API, and the target Namespace.
+
+The authorization and claim mapping logic is customizable, making it available to a variety of use cases and identity schemes.
+
```

Comment:
suggest including @sergeybykov's system diagram here as it was quite helpful

Reply:
I added a diagram for the order of operations.
Ya a video would be cool.